src/ott/solvers/quadratic/gromov_wasserstein.py

```diff
@@ -157,6 +157,9 @@ class GromovWasserstein(was_solver.WassersteinSolver):
     warm_start: Whether to initialize (low-rank) Sinkhorn calls using values
       from the previous iteration. If `None`, warm starts are not used for
       standard Sinkhorn, but used for low-rank Sinkhorn.
+    unscale_last_linearization: Whether to remove any scaling from the
+      cost matrices of the last linearized
+      :attr:`~ott.solvers.quadratic.gromov_wasserstein.GWOutput.geom`
```

Comment:
Maybe add for clarity:
unscale_last_linearization: Whether to remove any scaling from the
      cost matrices of the last linearization stored in
      :attr:`~ott.solvers.quadratic.gromov_wasserstein.GWOutput.geom`. This has the practical benefit that, while the OT coupling matrices obtained with GW might have been computed by re-scaling cost matrices for numerical stability, the last linearization stored in :attr:`~ott.solvers.quadratic.gromov_wasserstein.GWOutput.geom` will be un-scaled and recomputed with the original cost values.

Reply:
Done!